test/test_server.js

```diff
@@ -84,10 +84,21 @@ TestServer.start = function (config, printLogs) {
   return d.promise
 }
 
+function isDebug(){
+  return global.v8debug ? true : false
+}
+
 TestServer.prototype.start = function () {
+  var spawnOptions = ['./key_server_stub.js']
+
+  var nextDebugPort = process.debugPort + 2
+  if (isDebug()) {
+    spawnOptions.unshift('--debug-brk=' + nextDebugPort)
```

Comment:
@vbudhram do you need to log which debug port it is using? or does WebStorm figure it out?